src/utilities/number.js

```diff
@@ -7,3 +7,10 @@ export const getMiddleIndex = (number: string): number => {
 
   return isOdd(number) ? middle : middle - 1
 }
+
+export const formatNumber = num => {
+  if (num === null || num === undefined) {
```

Comment:
@plbabin There's a `isDefined` util from `utilities/is` that checks for this :D

Reply:
If you need a number, you can also check for `isNumber()`

Reply:
oh nice!! I updated the util in 85d5c88a